CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Fixed
+- Bump `full_moon_derive`'s edition to `2021`.
```

Comment:
I don't really consider this a user facing change, so we don't need a changelog entry